src/similarity.ts

```diff
@@ -96,6 +98,22 @@ function countBodyDifferences(
       contentType2,
       rewriteBeforeDiffRules,
     );
+  } else if (contentType.startsWith("application/grpc-web-text")) {
+    return countBodyDifferencesGrpcWebText(
+      request1,
+      contentType1,
+      request2,
+      contentType2,
+      rewriteBeforeDiffRules,
+    );
+  } else if (contentType.startsWith("application/grpc-web")) {
+    return countBodyDifferencesGrpcWeb(
+      request1,
+      contentType1,
+      request2,
+      contentType2,
+      rewriteBeforeDiffRules,
+    );
```

Comment:
Adds special similarity scoring logic for gRPC-Web request types, like we do for other content types like JSON.